tests/SelfInitializingFakes.Tests.Api/ApprovedApi/SelfInitializingFakes/net5.0.approved.txt

```diff
@@ -1,6 +1,6 @@
 ﻿[assembly: System.Runtime.InteropServices.ComVisible(false)]
 [assembly: System.Runtime.InteropServices.Guid("08462e29-25c0-4a1f-9275-23b7898e95f0")]
-[assembly: System.Runtime.Versioning.TargetFramework(".NETStandard,Version=v1.6", FrameworkDisplayName="")]
+[assembly: System.Runtime.Versioning.TargetFramework(".NETStandard,Version=v5.0", FrameworkDisplayName="")]

```

Comment:
I realize you didn't manually write this, but... .NET Standard 5.0? I don't think this target framework exists!

Reply:
That's weird... but they run on AppVeyor? They should have failed, then, shouldn't they? Since the `received` and `approved` files would have been different...